index.js

```diff
@@ -86,7 +86,8 @@ const popupMenu = [
   },
 ];
 
-const test = popupMenu.map((item) => `
+const test = popupMenu.map(
+  (item) => `
    <button rel='noopener noreferrer' href='javascript:void(0)' id='close-modal'>&times;</button>
   <div class='modal-heading'>
         <h2>${item.title}</h2>
```

Comment:
- [ ] Great job so far, however, there're few things you need to correct before your project could be approved.

-    You're required to remove already created HTML code for projects list in the `index.html`, and instead it should be added via JavaScript. and all of the HTML in that section is created dynamically when the page loads. No code for projects list should remain in the `index.html` again and instead should be created dynamically from your javascript file when the page is loaded.
- The popup window, both the desktop and the mobile must stick to the figma design. Kindly check the two screenshots of your design and the figma and work your alongside the figma design. At  this mement, all the information on your popup window is not visible on the screen when it opens. Kindly look into that  and fix.

### Your Design Desktop
[your-desktop-popup.webm](https://user-images.githubusercontent.com/26597432/224278084-72d598ec-71b7-49a8-8c3f-c35aed27c250.webm)

### Figma Design Desktop
![temp-desktop-popup](https://user-images.githubusercontent.com/26597432/224278141-ac1f1357-d89d-4d38-bb14-02a8593719eb.png)


### Your Design Mobile
[your-mobile-popup.webm](https://user-images.githubusercontent.com/26597432/224278189-722a9ad1-328e-480b-80cf-eedcb3d8e837.webm)

### Figma Design Mobile
![temp-mobile-popup](https://user-images.githubusercontent.com/26597432/224278238-36ce6066-a554-4e4d-9dfc-f4b8b2ae8b0e.png)